docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc

```diff
@@ -112,6 +125,8 @@ Adjust the docker-compose file above by adding these environment variables:
       - FLEET_SERVER_SERVICE_TOKEN={service-token}
 ----
 
+NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. See {observability-guide}/synthetics-quickstart-fleet[Synthetics Fleet Quickstart] for more information.
```

Comment:
```suggestion
NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. See {observability-guide}/synthetics-quickstart-fleet.html[Synthetics Fleet Quickstart] for more information.
```